libcontainer/intelrdt/intelrdt.go

```diff
@@ -76,7 +97,8 @@ import (
  *     |-- tasks
```

Comment:
```
+ *     |-- mon_data
+ *         |-- mon_L3_00
+ *             |-- llc_occupancy
+ *             |-- mbm_local_bytes
+ *             |-- mbm_total_bytes
+ *         |-- 
```

Add the tree hierarchy of  `/sys/fs/resctrl/container_id/mon_data` accordingly for monitoring in CTRL_MON group.